File: code/cs/OpcReadNode.cs
// DOC
OpcValue isRunning = client.ReadNode("ns=2;s=Machine/IsRunning");

// DOC
OpcReadNode[] commands = new OpcReadNode[] {
    new OpcReadNode("ns=2;s=Machine/Job/Number"),
    new OpcReadNode("ns=2;s=Machine/Job/Name"),
    new OpcReadNode("ns=2;s=Machine/Job/Speed")
};

IEnumerable<OpcValue> job = client.ReadNodes(commands);

// DOC
OpcValue isRunningDisplayName = client.ReadNode("ns=2;s=Machine/IsRunning", OpcAttribute.DisplayName);

// DOC
OpcReadNode[] commands = new OpcReadNode[] {
    new OpcReadNode("ns=2;s=Machine/Job/Number", OpcAttribute.DisplayName),
    new OpcReadNode("ns=2;s=Machine/Job/Name", OpcAttribute.DisplayName),
    new OpcReadNode("ns=2;s=Machine/Job/Speed", OpcAttribute.DisplayName)
};
 
IEnumerable<OpcValue> jobDisplayNames = client.ReadNodes(commands);

File: code/vb/OpcReadNode.vb
'DOC
Dim isRunning As OpcValue = client.ReadNode("ns=2;s=Machine/IsRunning")

'DOC
Dim commands As OpcReadNode() = New OpcReadNode() { _
        New OpcReadNode("ns=2;s=Machine/Job/Number"), _
        New OpcReadNode("ns=2;s=Machine/Job/Name"), _
        New OpcReadNode("ns=2;s=Machine/Job/Speed") _
}

Dim job As IEnumerable(Of OpcValue) = client.ReadNodes(commands)

'DOC
Dim isRunningDisplayName As OpcValue = client.ReadNode("ns=2;s=Machine/IsRunning", OpcAttribute.DisplayName)

'DOC
Dim commands As OpcReadNode() = New OpcReadNode() { _
        New OpcReadNode("ns=2;s=Machine/Job/Number", OpcAttribute.DisplayName), _
        New OpcReadNode("ns=2;s=Machine/Job/Name", OpcAttribute.DisplayName), _
        New OpcReadNode("ns=2;s=Machine/Job/Speed", OpcAttribute.DisplayName) _
}

Dim jobDisplayNames As IEnumerable(Of OpcValue) = client.ReadNodes(commands)
